extra_model/_vectorizer.py

```diff
@@ -24,7 +24,10 @@ def __init__(self, embedding_file, raw=True):
             #  normalized ones = saves lots of memory!"
             self.wv_glove.init_sims()
         else:
-            glove_file = datapath("/app/glove.840B.300d.txt")
+            if embedding_file is None:
```

Comment:
It was a mistake on my part to push all of this `raw` business into main since I only needed it to test how long it takes to load embeddings from `txt` vs loading binary. Since we will always load binary (the difference is 10x in favor of that), I'd say we should delete the whole thing and return it back to how it was here - https://github.csnzoo.com/shared/extra-model/blob/master/extra_model/Vectorizer.py
I've already have a way to support different embeddings in tests (here is an example - https://github.csnzoo.com/shared/extra-model/blob/master/tests/test_extra_model.py), so there is truly no need.